project_config.yml

```
@@ -10,7 +10,11 @@ num_features:
 cat_features:
-  - dropoff_zip
   - pickup_zip
+  - dropoff_zip
+
+target: fare_amount
+
+id_col: pickup_zip
 
-target: fare_amount
\ No newline at end of file
+mlflow_experiment_name: /Shared/mlops_course_annika
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)